zazu/util.py

```diff
@@ -44,13 +49,15 @@ def __init__(self, **entries):
 
 
 def check_output(*args, **kwargs):
+    """Like subprocess.check_output but raises an exception if it cannot be found."""
     try:
         return subprocess.check_output(*args, **kwargs)
     except OSError:
         raise_uninstalled(args[0][0])
 
 
 def call(*args, **kwargs):
+    """Like subprocess.call but raise an exception if it cannot be found."""
     try:
         return subprocess.call(*args, **kwargs)
```

Comment:
```subprocess.check_call()```
I believe could help broaden the call check.

Something like:
```
    try:
       subprocess.check_call(['someCall])
    except subprocess.CalledProcessError:
       return whatever
```
I think you can get away without try catch too.